quiche/balsa/balsa_frame.h

```diff
@@ -226,7 +226,7 @@ class QUICHE_EXPORT BalsaFrame : public FramerInterface {
   }
 
   // If enabled, parse the available portion of headers even on a
-  // HEADERS_TOO_LONG error, so that that portion of headers is available to the
+  // HEADERS_TOO_LONG error, so that portion of headers is available to the
```

Comment:
Sometimes `that that` is meaningful. In this case `that portion` should work just as well as `that that portion` even though there's technically a change in the parsing.